ui/css/base/_colors.scss

```diff
@@ -56,3 +56,4 @@ $color-shadow:               rgba(#000, 0.3) !default;
 
 // New pallet
 $pallet-blue:                #1067a6 !default;
```

Comment:
I like this pallet variable set. Just so you know, this blue is the same as `$color-primary-darker`.

Reply:
Perhaps in your branch? `$color-primary-darker` is [currently](https://github.com/18F/omb-eregs/blob/8f1dd46cfd1f392ea544eeb88cb7f9b72a075c38/ui/css/base/_colors.scss#L5) `#3d73a8`.

Reply:
I like using the `$color` variables, but we need to figure out a strategy for them. See #648